<a href="https://colab.research.google.com/github/gimseng/learnML/blob/master/Titanic.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [0]:
# Import the usual suspects
import numpy as np 
import pandas as pd 
import seaborn as sns
%matplotlib inline
from matplotlib import pyplot as plt
from matplotlib import style

In [2]:
#I uploaded the train.csv and test.csv (from Kaggle) to my google drive. Run this and then authorize it appropriately
# Then the pd.read_csv after these codes will read them. For me, I put everything in Colab Notebooks/Titanic/
from google.colab import drive
drive.mount('/content/drive')

Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount("/content/drive", force_remount=True).


In [0]:
test = pd.read_csv("/content/drive/My Drive/Colab Notebooks/Titanic/test.csv")
train = pd.read_csv("/content/drive/My Drive/Colab Notebooks/Titanic/train.csv")

In [8]:
test.head()

PassengerId  Pclass  ... Cabin Embarked
0          892       3  ...   NaN        Q
1          893       3  ...   NaN        S
2          894       2  ...   NaN        Q
3          895       3  ...   NaN        S
4          896       3  ...   NaN        S

[5 rows x 11 columns]

In [9]:
train.head()

PassengerId  Survived  Pclass  ...     Fare Cabin  Embarked
0            1         0       3  ...   7.2500   NaN         S
1            2         1       1  ...  71.2833   C85         C
2            3         1       3  ...   7.9250   NaN         S
3            4         1       1  ...  53.1000  C123         S
4            5         0       3  ...   8.0500   NaN         S

[5 rows x 12 columns]

In [11]:
train.shape, test.shape

((891, 12), (418, 11))

(<bound method DataFrame.info of      PassengerId  Survived  Pclass  ...     Fare Cabin  Embarked
 0              1         0       3  ...   7.2500   NaN         S
 1              2         1       1  ...  71.2833   C85         C
 2              3         1       3  ...   7.9250   NaN         S
 3              4         1       1  ...  53.1000  C123         S
 4              5         0       3  ...   8.0500   NaN         S
 ..           ...       ...     ...  ...      ...   ...       ...
 886          887         0       2  ...  13.0000   NaN         S
 887          888         1       1  ...  30.0000   B42         S
 888          889         0       3  ...  23.4500   NaN         S
 889          890         1       1  ...  30.0000  C148         C
 890          891         0       3  ...   7.7500   NaN         Q
 
 [891 rows x 12 columns]>,
 <bound method DataFrame.info of      PassengerId  Pclass  ... Cabin Embarked
 0            892       3  ...   NaN        Q
 1            893      

Thus we have imported the two csv files correcly. There are 891 passengers for the training set while there are 418 passengers for the test set. Of course the test set does not have the predicted value "Survived" or not.